sdk/appcenter/src/main/java/com/microsoft/appcenter/http/DefaultHttpClientCallTask.java

```diff
@@ -175,7 +175,7 @@ private String doHttpCall() throws Exception {
              * Don't hardcode TLS version when enabled by default to avoid unnecessary wrapping and
              * to support future versions of TLS such as say 1.3 without having to patch this code.
              */
-            if (Build.VERSION.SDK_INT < Build.VERSION_CODES.KITKAT_WATCH) {
+            if (Build.VERSION.SDK_INT <= Build.VERSION_CODES.LOLLIPOP) {
```

Comment:
It should be `< LOLLIPOP`, not `<=`, TLS 1.2 was enabled on 5.0: https://developer.android.com/about/versions/android-5.0-changes#ssl.

What is weird is that API level 19 is already covered by this patch (kitkat), [kitkat watch is API level 20](https://developer.android.com/reference/android/os/Build.VERSION_CODES#KITKAT_WATCH), not 19.
TLS 1.2 should have been enabled on API level 20 according to https://developer.android.com/reference/javax/net/ssl/SSLSocket#default-configuration-for-different-android-versions.

Have you tried to repro the error?

Reply:
Based on https://github.com/square/okhttp/issues/2372#issuecomment-244807676
> the same issue also happens on some Samsung devices with API 21